src/daemon/http/server.rs

```diff
@@ -1168,6 +1169,19 @@ async fn api_ca_stats(req: Request, path: &mut RequestPath, ca: Handle) -> Routi
     }
 }
 
+async fn api_ca_sync(req: Request, path: &mut RequestPath, ca: Handle) -> RoutingResult {
+    if req.is_post() {
+        let actor = req.actor();
```

Comment:
This currently only checks at the parent level if the caller has CA_READ permission. Is that a suitable permission for something that has side-effects like sync or should we enforce a higher permission requirement here?

Reply:
Changed to require CA_UPDATE. It doesn't really matter because this can't break anything, but still.